@navikt/ds-react/accordion/src/Accordion.tsx

```diff
@@ -56,7 +57,7 @@ const Accordion = forwardRef<HTMLDivElement, AccordionProps>(
           onClick={onClick ? onClick : () => setInternalOpen((open) => !open)}
           {...rest}
         >
-          <span className="navds-accordion__title">{title}</span>
+          <span className="navds-accordion__title">{headline}</span>
```

Comment:
Jeg tenker det burde være samsvar mellom klassenavnet og variabelnavnet til innholdet her, for å gjør koden mer konsekvent. Mitt forslag er `<span className="navds-accordion__heading">{heading}</span>`, men headline funker jo også.